src/docs/debug-performance.md

```diff
@@ -54,7 +54,7 @@ Benchmark (Aggregate): Template Write took 2088ms (27.6%, called 312×, 6.7ms ea
 
 Note that while we do make every attempt to make these as accurate as possible, the percentages for these entries may be greater than 100% due to the asynchronous nature of these tasks (passthrough copy especially).
 
-## Node.js Profiling
+## Node.js CPU Profiling <span id="node-js-profiling"></span>
```

Comment:
Since you're using Markdown-It, this could be eased with a [nifty plugin](https://www.npmjs.com/package/markdown-it-attrs).

Reply:
This `id` is specifically for backwards compat with the old hash